modules/installation-infrastructure-user-infra.adoc

```diff
@@ -35,6 +42,9 @@ endif::ibm-z[]
 ifndef::ibm-z[]
 . Configure DHCP or set static IP addresses on each node.
 endif::ibm-z[]
+ifdef::ibm-z-kvm[]
```

Comment:
@jboxman 
Trying to figure out how to best approach your comment how about changing this too: 
`. Choose to perform a either a fast track installation of {op-system-first}  or a full installation {op-system-first} For the full installation you must set up an HTTP or HTTPS server to provide Ignition files to the cluster nodes. This is not required for a fast track installation. See sections “Creating {op-system-first} machines - fast track installation" and “Creating {op-system-first} machines - full installation".`

And then add a two separate  modules to the KVM assembly.

Reply:
@SNiemann15, sure that seems reasonable, thanks!